spec/zoneddatetime.html

```diff
@@ -1115,11 +1137,19 @@ <h1>InterpretISODateTimeOffset ( _year_, _month_, _day_, _hour_, _minute_, _seco
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-totemporalzoneddatetime" aoid="ToTemporalZonedDateTime">
-      <h1>ToTemporalZonedDateTime ( _item_ [ , _options_ ] )</h1>
-      <p>
-        The abstract operation ToTemporalZonedDateTime returns its argument _item_ if it is already a Temporal.ZonedDateTime instance, converts _item_ to a new Temporal.PlainDate instance if possible, and throws otherwise.
-      </p>
+    <emu-clause id="sec-temporal-totemporalzoneddatetime" type="abstract operation">
+      <h1>
+        ToTemporalZonedDateTime (
+          _item_: an ECMAScript language value,
+          optional _options_: an Object,
+        )
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It returns its argument _item_ if it is already a Temporal.ZonedDateTime instance, converts _item_ to a new Temporal.PlainDate instance if possible, and throws otherwise.
```

Comment:
```suggestion
          It returns its argument _item_ if it is already a Temporal.ZonedDateTime instance, converts _item_ to a new Temporal.ZonedDateTime instance if possible, and throws otherwise.
```
Optional, but we may as well fix this while you're in here anyway :smile:

Reply:
Done :+1: